django_cyverse_auth/authBackends.py

```diff
@@ -371,8 +372,9 @@ def keystone_validate_token(self, auth_url, username, token, project_name, proje
         auth_token = ks_session.get_token()
         expiry_time = token_auth.auth_ref.expires
         try:
+            email = self._lookup_email(session)
             self._keystone_auth_to_token(token_auth, username, project_name)
```

Comment:
`token` in the next line will be the same value passed in as a parameter.. Based on L355 I think you're missing a `token =` here?